wgpu-hal/src/dx12/adapter.rs

```diff
@@ -529,8 +549,8 @@ impl super::Adapter {
                     // Direct3D correctly bounds-checks all array accesses:
                     // https://microsoft.github.io/DirectX-Specs/d3d/archive/D3D11_3_FunctionalSpec.htm#18.6.8.2%20Device%20Memory%20Reads
                     uniform_bounds_check_alignment: wgt::BufferSize::new(1).unwrap(),
-                    raw_tlas_instance_size: 0,
-                    ray_tracing_scratch_buffer_alignment: 0,
+                    raw_tlas_instance_size: 64,
+                    ray_tracing_scratch_buffer_alignment: 256,
```

Comment:
Are these d3d12 constants? If so can they be referenced directly?

Reply:
The first one is a size, I can use mem::size_of. The second one does have a constant.